source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,204 @@
+standards. Below are some examples: 
+
+* `RC4`, `SSLv3`, `3DES`, and `SHA-1` ciphers which have had known 
+  vulnerabilities, have been explicitly disabled within many |CL| packages to 
+  avoid their accidental usage. 
+
+* Services and subsystems which expose sensitive system information 
+  have been removed such as the `finger` and `tcpwrappers`.
+
+* `SFTP` has been disabled by default due to security 
+  considerations. See the `openssh-server reference page`_ for more details. 
+
+
+Verified trust
+--------------
+
+|CL| encourages the use of secure practices such as encryption
+and digital signature verification throughout the system and discourages blind
+trust. Below are some examples: 
+
+* All update operations from swupd are transparently encrypted and checked 
+  against the |CL| maintainers' public key for authenticity. 
+  More information can be found in this blog post: 
+  `blog post about swupd security`_ 
+
+* Before being built, packages available from |CL| verify checksums and 
+  signatures provided by third party project codebases and maintainers.
+
+* |CL| features a unified certificate store, `clrtrust`_ which comes 
+  ready to work with well-known Certificate Authorities out of the box. 
+  `clrtrust`_ also offers an easy to use command line interface for managing 
+  system-wide chains of trust, instead of ignoring foreign certificates. 
+
+
+ 
+
+
+
+Compiled with secure options
+----------------------------
+
+While |CL| packages are optimized for performance on 
+Intel® architecture, security conscious kernel and compiler options are 
+sensibly taken advantage of. Below are some examples: 
+
+ 
+* Kernels shipped with Clear Linux are signed and disallow the usage of 
+  custom kernel modules to maintain verifiable system integrity.
+
+* `Address space layout randomization (ASLR)`_ and 
+  `Kernel address space layout randomization (KASLR)`_  are kernel features
+  which defend against certain memory based attacks. 
+  More information can be found in a `blog post about PIE executables`_ .
+
+* `dm-verity`_ is a kernel mechanism readily available in |CL| 
+  which verifies integrity of the devices being written to, like hard disks,
+  to help ensure they have not been tampered with.  
+
+
+
+  
+
+Security in System Design
+=========================
+
+Simple, yet effective, techniques are used throughout the 
+|CL| system design to defend against common attack vectors and enable
+good security hygiene. Below are some examples: 
+
+
+* Full disk encryption using `Linux Unified Key Setup`_ (LUKS)  is available 
+  during installation.
+
+* |CL| uses the PAM cracklib module to harden user login and password
+  security resulting in: 
+
+  - No default username or root password set out of the box with 
+    |CL|, you will be asked to set your own password immediately.
+
+  - Simple password schemes, which are known to be easily compromised,
+   cannot be set in |CL|.
```

Comment:
@puneetse. If you can, please change second line in a sub-item to left-align with the line above it for this and the next sub-item.  (IOW: Move "cannot" one space to the right. ;)